content/docs/getting-started/fluvio-cloud.md

```diff
@@ -23,17 +23,17 @@ Head on over to the [Fluvio Cloud signup page] to create an account.
 
 [Fluvio Cloud signup page]: https://cloud.fluvio.io/signup
 
-![Fluvio Cloud signup](/img/getting-started/cloud-signup.png)
+{{< image src="getting-started/cloud-signup.png" alt="Fluvio Cloud signup" justify="center" width="400" type="scaled-75" >}}
 
 After filling out the form, you'll be greeted with a success message telling
 you to verify your email. You'll need to complete this step in order to
 continue.
 
-![Fluvio Cloud verification](/img/getting-started/cloud-verification.png)
+{{< image src="getting-started/cloud-verification.png" alt="Fluvio Cloud verification" justify="center" width="600" type="scaled-75" >}}
```

Comment:
since this a wider window, you could also you different scaling percentages:
  img#scaled-98 {width: 98%;}
  img#scaled-95 {width: 95%;}
  img#scaled-90 {width: 90%;}
  img#scaled-80 {width: 80%;}
  img#scaled-75 {width: 75%;}
  img#scaled-60 {width: 60%;}
  img#scaled-50 {width: 50%;}

Reply:
To test just shrink the window.

Reply:
How do the scalings behave? I'm not really familiar with it